### PR TITLE
Update remove-licenses-from-users.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a line about retention and what happens to items when the mailbox is deleted.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6600

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/manage/remove-licenses-from-users.md [hunk 1]
```diff
@@ -131,7 +131,7 @@ When you use the **Active users** page to unassign licenses, you unassign produc
 
 ## What happens to a user's data when you remove their license?
 
-- When a license is removed from a user, Exchange online data that is associated with that account is held for 30 days. After the 30-day grace period, the data is deleted and can't be recovered.
+- When a license is removed from a user, Exchange online data that is associated with that account is held for 30 days. After the 30-day grace period, the data is deleted and can't be recovered. However, It adheres to retention policy and the content that matches retention labels are retained for discovery.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/remove-licenses-from-users.md]:
```suggestion
- When a license is removed from a user, Exchange Online data that is associated with that account is held for 30 days. After the 30-day grace period, the data is deleted and can't be recovered. However, it is linked to the retention policy, and the content that matches retention labels is retained for discovery.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joshfolcik** [comment]
> Added a line about retention and what happens to items when the mailbox is deleted.

@msbemba Your update specifically mentions retention labels, does that include a retention policy (not a user label but a policy in the compliance center for retention such as one that does this: "Keep all email on all mailboxes for 2 years from create and then do nothing" such as what we have, users can also choose longer using retention labels on individual items/folders.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@cmcatee-MSFT Gentle reminder

**Pull request merged**